setup.py

```diff
@@ -1,15 +1,31 @@
 #!/usr/bin/env python
-
 from setuptools import setup
 
 
-setup(name="pubsubsuperfeedr",
-      version="0.3.2",
-      description="Library for working with Superfeedr's PubSubHubbub API",
-      author="Urban Airship",
-      author_email="contact@urbanairship.com",
-      url="http://github.com/urbanairship/pubsubsuperfeedr/",
-      py_modules=["pubsubsuperfeedr"],
-      install_requires=["feedparser", "python-dateutil"],
-      license="MIT License",
+setup(
+    name="pubsubsuperfeedr",
+    version="0.3.2",
+    description="Library for working with Superfeedr's PubSubHubbub API",
+    long_description=open('README.rst').read(),
+    author="Urban Airship",
+    author_email="contact@urbanairship.com",
```

Comment:
Honestly not sure who this would even _go_ to.

Reply:
i'll change it to the same as our npm modules, which i believe is web-team@urbanairship.com